### DISCUSSION TITLE
Best settings for best battery life

### DISCUSSION THREAD

**Discussion by amansinghaljpr** [seerge/g-helper#1804] [Q&A]
I was wondering , if g-helper offers us the best possible performance out of your gaming laptop . Can we tune it in a way to get the best battery life out of it . 
I have a 2023 G14 with R9 , and rtx4060 . I believe on the eco mode the rx780M can handle daily browsing, documents etc with pretty good efficiency. But if i have to get the best battery life out of this laptop as it uses a ryzen 7000 series CPU what settings should i choose which can make it comparable to an ultrabook .

**Comment by seerge** [comment]
I would say next to turning off dGPU which you already did, second most important  thing would be making sure that windows is not running anything unnecessarily :) so set Battery Saver ON there, it usually stops / postpones some background jobs.

**Comment by Kikimahe** [comment]
hi, 

On a gaming laptop, it will be difficult to obtain the same battery life as an ultrabook. ;-)
But you can try changing some settings to optimize it.

**In Ghelper**
- Use CPU Silent mode and disable "CPU Boost" on silent mode
- Apply a CPU limit (Use HWinfo Max CPU value when doing simple tasks to adjust the limit)
- GPU in ECO mode (of course).

**In Windows:**
- Screen lighting at minimum
- Static keyboard lighting at minimum or disabled.
- In device manager, disable the Wifi/Bluetooth and USB ports, if you don't need them.
- And as Seerge said, remove all processes/software that may be running in the background. Also disable anything related to Windows telemetry that could be running in the background. (google it)
- In the task manager you can also use Windows efficiency mode on a process.

Good Luck ;-)
